docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -82,214 +78,348 @@ To enable your application to use the image util functionality:
-   ret = image_util_transform_create(&handle);
+   transformation_h transform_h;
+   ret = image_util_transform_create(&transform_h);
    ```
 
-2. Set the target resolution using the `image_util_transform_set_resolution()` function:
+2. Set the target resolution using `image_util_transform_set_resolution()`:
 
    ```
-   ret = image_util_transform_set_resolution(handle, width, height);
+   ret = image_util_transform_set_resolution(transform_h, width, height);
    ```
 
-3. Execute the transformation using the `image_util_transform_run()` function:
+3. Run the transformation using `image_util_transform_run2()` or `image_util_transform_run2_async()`:
 
    ```
-   ret = image_util_transform_run(handle, src,
-                                  (image_util_transform_completed_cb)completed_callback,
-                                  user_data);
+   ret = image_util_transform_run2(transform_h, src_image, &dst_image);
    ```
 
    > **Note**
    >
    > - The image format has no effect on the transformation.
-   > - If the color space is YUV, the target image width and height must be multiples of 8. This restriction does not apply to RGB images.
-   > - If hardware acceleration is enabled, you can execute 2 more image transformations using the same transformation handle.
+   > - If the color space is YUV, the width and height of the target image must be multiples of 8. This restriction does not apply to the RGB images.
+
+4. If `image_util_transform_run2_async()` is used to run the transformation, handle the transformation results in the `image_util_transform_completed2_cb()` callback, which is invoked after the transformation is complete.
+
+5. After the transformation is complete, destroy the transformation handle using `image_util_transform_destroy()`:
+
+   ```
+   ret = image_util_transform_destroy(transform_h);
+   ```
+
+<a name="resize_mediapacket"></a>
+## Resizing Media Packet
+
+To resize a media packet:
+
+1. Create a transformation handle using `image_util_transform_create()`:
+
+   ```
+   transformation_h transform_h;
+   ret = image_util_transform_create(&transform_h);
+   ```
+
+2. Set the target resolution using `image_util_transform_set_resolution()`:
+
+   ```
+   ret = image_util_transform_set_resolution(transform_h, width, height);
+   ```
+
+3. Run the transformation using `image_util_transform_run()`:
+
+   ```
+   ret = image_util_transform_run(transform_h, (media_packet_h)src_packet,
+                                (image_util_transform_completed_cb)completed_callback,
+                                user_data);
+   ```
+
+   > **Note**
+   >
+   > - The image format has no effect on the transformation. If the color space is YUV, then the width and height of the target image must be multiples of 8. This restriction does not apply to the RGB images.
 
 4. Handle the transformation results in the `image_util_transform_completed_cb()` callback, which is invoked after the transformation is complete.
```

Comment:
Suggest removing the and callback. Here in the `image_util_transform_completed_cb()`, cb denotes callback. Therefore, suggest rephrasing to:
Handle the transformation results in `image_util_transform_completed_cb()` , which is invoked after the transformation is complete.